gridData/core.py

```diff
@@ -546,6 +553,18 @@ def _export_dx(self, filename, type=None, typequote='"', **kwargs):
         dx = OpenDX.field('density', components=components, comments=comments)
         dx.write(filename)
 
+    def _export_dxgz(self, filename, type=None, typequote='"', **kwargs):
```

Comment:
Too much code duplication, use the filename extension logic in `_export_dx()`. (Can later be replaced with `openany(filename, "w")`.)